modules/sdk-coin-xlm/src/xlm.ts

```diff
@@ -1042,9 +1043,11 @@ export class Xlm extends BaseCoin {
         throw new Error('keychains are required, but could not be fetched');
       }
 
+      assert(keychains.backup.pub);
```

Comment:
why would only xlm need to be changed for this PR? is this our only onchain multisig coin?

Reply:
Apparently it was the only coin that did something with that pub that no longer works if it can be undefined.